app/src/containers/ListOfNews/index.js

```diff
@@ -45,27 +43,30 @@ class ListOfNews extends React.Component {
     }
 
     getNewsListAPI(callback) {
-        const url = new URL(window.location.protocol + '//' + window.location.hostname + this.state.portOption + "/api/v1/news");
-
+      const url = new URL(window.location.protocol + '//' + window.location.hostname + this.state.portOption + "/api/v1/news");
         fetch(url, {
             method: 'GET',
             credentials: 'include',
             cors: 'no-cors'
         }).then((response) => response.json())
             .then((responseJson) => {
+                responseJson.sort((a,b) => new Date(a.updatedAt) < new Date(b.updatedAt) ? 1 : -1);
```

Comment:
This should be the only change of the pr but all the other changes are either yours or for your local setup. Please do not include changes that you have done for local run.

I am not very satisfied with the sorting function logic. It is near unreadable. We should avoid writing complex logic like this which does not possess readability. Also, instead of just inlining the function you can put that function in a helper file. We do not have any utility helper file so you can go ahead and create that and put the function there. You need to think about what are considered as utility helpers. Simply put, logic/helpers that are not related to the business logic. Here you will be doing a compare between two date so this can be considered as a util. How about the following function,
In the `util.js`:
`const compareDate = (firstDate, secondDate) => (new Date(secondDate) - new Date(firstDate))`

Then, you do not have to write extra line for sorting. You could simply just do `this.setState({ newsList: responseJson.sort((item, siblingItem) => compareDate(item.updatedAt, siblingItem.updatedAt))})`